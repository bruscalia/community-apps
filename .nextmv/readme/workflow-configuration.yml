apps:
  - name: cost-flow-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-ampl
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-gosdk
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-gurobi
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-java-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: knapsack-pyomo
    scripts:
      - name: 0.sh
        silent: true
  - name: nextroute
    scripts:
      - name: 0.sh
        silent: true
      - name: 1.sh
        skip: true
  - name: order-fulfillment-gosdk
    scripts:
      - name: 0.sh
        silent: true
  - name: routing-java-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: routing-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: shift-assignment-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: shift-assignment-pyomo
    scripts:
      - name: 0.sh
        silent: true
  - name: shift-planning-ortools
    scripts:
      - name: 0.sh
        silent: true
  - name: shift-planning-pyomo
    scripts:
      - name: 0.sh
        silent: true
  - name: shift-scheduling-gosdk
    scripts:
      - name: 0.sh
        silent: true
  - name: xpress
    scripts:
      - name: 0.sh
        silent: true
