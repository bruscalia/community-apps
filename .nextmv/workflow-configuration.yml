apps:
  - name: knapsack-gosdk
    type: go
    sdk_version: latest
    app_id: knapsack-gosdk
    marketplace_app_id: knapsack.gosdk
  - name: knapsack-java-ortools
    type: java
    app_id: knapsack-jortools
    marketplace_app_id: knapsack.jortools
  - name: knapsack-ortools
    type: python
    app_id: knapsack-ortools
    marketplace_app_id: knapsack.ortools
  - name: knapsack-pyomo
    type: python
    app_id: knapsack-pyomo
    marketplace_app_id: knapsack.pyomo
  - name: nextroute
    type: go
    sdk_version: latest
    app_id: nextroute
    marketplace_app_id: nextroute
  - name: order-fulfillment-gosdk
    type: go
    sdk_version: latest
    app_id: order-fulfill-gosdk
    marketplace_app_id: order.fulfill.gosdk
  - name: routing-ortools
    type: python
    app_id: routing-ortools
    marketplace_app_id: routing.ortools
  - name: shift-assignment-ortools
    type: python
    app_id: shift-assign-ortools
    marketplace_app_id: shift.assign.ortools
  - name: shift-assignment-pyomo
    type: python
    app_id: shift-assign-pyomo
    marketplace_app_id: shift.assign.pyomo
  - name: shift-planning-ortools
    type: python
    app_id: shift-plan-ortools
    marketplace_app_id: shift.plan.ortools
  - name: shift-planning-pyomo
    type: python
    app_id: shift-plan-pyomo
    marketplace_app_id: shift.plan.pyomo
  - name: shift-scheduling-gosdk
    type: go
    sdk_version: latest
    app_id: shift-scheduling
    marketplace_app_id: shift.scheduling
  - name: xpress
    type: python
    app_id:
    marketplace_app_id:
  - name: cost-flow-ortools
    type: python
    app_id:
    marketplace_app_id:
