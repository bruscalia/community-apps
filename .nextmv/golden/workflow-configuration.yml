apps:
  - name: cost-flow-ortools
    language: python
    language-version: 3.11
  - name: knapsack-ampl
    language: python
    language-version: 3.11
  - name: knapsack-gosdk
    language: go
    language-version: 1.22
  - name: knapsack-gurobi
    language: python
    language-version: 3.11
  - name: knapsack-java-ortools
    language: java
    language-version: 21
    distribution: temurin
  - name: knapsack-ortools
    language: python
    language-version: 3.11
  - name: knapsack-pyomo
    language: python
    language-version: 3.11
  - name: nextroute
    language: go
    language-version: 1.22
  - name: order-fulfillment-gosdk
    language: go
    language-version: 1.22
  - name: routing-ortools
    language: python
    language-version: 3.11
  - name: shift-assignment-ortools
    language: python
    language-version: 3.11
  - name: shift-assignment-pyomo
    language: python
    language-version: 3.11
  - name: shift-planning-ortools
    language: python
    language-version: 3.11
  - name: shift-planning-pyomo
    language: python
    language-version: 3.11
  - name: shift-scheduling-gosdk
    language: go
    language-version: 1.22
  - name: xpress
    language: python
    language-version: 3.11
